pilot/pkg/networking/core/v1alpha3/cluster.go

```diff
@@ -172,10 +248,11 @@ func (p clusterPatcher) hasPatches() bool {
 
 // SniDnat clusters do not have any TLS setting, as they simply forward traffic to upstream
 // All SniDnat clusters are internal services in the mesh.
+// TODO enable cache
```

Comment:
```suggestion
```

cache is enabled, right?

Reply:
Not for this part of the code. Only for buildOutboundClusters

Reply:
There is none other than trying to keep a big PR from being even bigger.

The sni dnat is only for a very small number of proxies so it's not a high priority at all. It may not even be effective to cache given it will take up space in the cache but is unlikely to cause cache hit

Reply:
I guessed this reason